resources/less/hakija.less

```diff
@@ -1010,11 +1010,31 @@ textarea.application__form-text-area__size-large {
   }
 }
 
-.application__form-checkbox {
-  display: none;
+input.application__form-checkbox {
+  appearance: none;
+  -webkit-appearance: none;
+  width: 1.15em;
+  height: 1.15em;
+  border-radius: 0.15em;
+  border-width: 1px;
+  margin-right: 10px;
+  box-shadow: inset 0 0 0 1px @neutral-grey-700;
+}
+
+input.application__form-checkbox:checked {
```

Comment:
Laita nämä sisäkkäin ylemmän määrittelyn kanssa käyttäen &:checked

Sama myös disabledille